client/Annotators/EDF/timeSeriesAnnotator.js

```diff
@@ -12711,6 +12713,8 @@ $.widget("crowdeeg.TimeSeriesAnnotator", {
       console.log(annotatorConfig);
       if(Object.keys(annotatorConfig.scalingFactors).length != Object.keys(that.vars.originalScalingFactors).length){
         window.alert("The preferences file you wish to upload is not compatible with the chart (number of channels do not match). Please choose another file.");
+      } else if (annotatorConfig.startTime < 0 || annotatorConfig.startTime > that.vars.recordingLengthInSeconds) {
```

Comment:
For both cases, it might be easier for the user if we just set the start time to 0 if it's invalid, let them know we changed it, and continue with _savePreferences(). The rest of the preferences might still be valuable, so we don't necessarily need to discard the entire file.